decred_wallet/Features/Security/PinInputView.swift

```diff
@@ -37,17 +38,14 @@ class PinInputView: UIView {
     }
     
     override func draw(_ frame: CGRect) {
+        // clear current views
```

Comment:
You may need to just take off this comment. Now it seem like it also apply to line 44.